spec/wordlist_repo_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+require 'ronin/wordlists/wordlist_repo'
+
+describe Ronin::Wordlists::WordlistRepo do
+  subject { described_class.new(path) }
+
+  let(:path) { File.expand_path(File.join(__dir__, '..', 'spec', 'fixtures', 'wordlists')) }
+
+  describe "#initialize" do
+    it "must initialize #path and #name" do
+      expect(subject.path).to eq(path)
+      expect(subject.name).to eq("wordlists")
+    end
+  end
+
+  describe "#download" do
+    let(:repo_path) { File.join(Dir.pwd, "") }
```

Comment:
I would probably use `/does/not/exist` here. If something accidentally get's downloaded, it will fail to write to `/does/not/exist` instead of downloading into `ronin-recon`.